extension/colour.js

```diff
@@ -0,0 +1,8 @@
+window.addEventListener("load", () => {
+	const masthead = document.getElementById("masthead");
+	if (masthead) {
```

Comment:
This if is here because SOMETIMES the header just... isn't there yet?

Like the document is loaded, but I think the header gets created dynamically, so I just miss it due to bad timing?

Reply:
OK turns out sometimes it was in a `<frame>` (which is like another website)

Fixed.